### PR TITLE
New back in/out animations added

### PR DESCRIPTION
**Pull request by ardagedikk**
https://codepen.io/ardagedikk/pen/Ewbjyx

### CODE REVIEW

**Comment by eltonmesquita** [comment]
Hello @ardagedikk.
I enjoyed the proposal and I think they have potential. There's one animation which I doesn't make sense: `twistInLeft`. It should be the opposite of `twistInRight`.
Can you fix it?
Also, can you please solve the conflicts so I can review it and hopefully merge it into master?

**Comment by ardagedikk** [comment]
Hello @eltonmesquita 
I fix it the animation and resolve conflicts , i think it was so much better now :) 
https://codepen.io/ardagedikk/pen/rGYaPP?editors=0100

**Comment by eltonmesquita** [comment]
Some points for improvement:
-The animations lack consistence between them. 
- Both entrances (TwistInDown and TwistInUp) start rotated already, maybe starting them with no rotation would be better.
- TwistInLeft and Right are "Out" not "In" animations and they behave differently from TwistInDown and TwistInUp. They also end up rotated, better finish them with no rotation. Try to fix these problems.
- I still have the feeling that they are a bit gratuitous. They need more work.

**Comment by eltonmesquita** [comment]
Any news on this @ardagedikk ?

**Comment by ardagedikk** [comment]
I'm sorry, I could not send updates because of the intensity. I've made a few improvements on the animations. I think they are more beautiful this way. 
Demo : https://codepen.io/ardagedikk/pen/Ewbjyx

**Comment by eltonmesquita** [comment]
To me the "linear" entrances and exit feel inconsistent. The previous version felt a bit more natural as they had more meaning (they started and ended with a clear meaning).

**Comment by ardagedikk** [comment]
I fixed the inconsistent parts in the entrances and exits. Maybe in the next update we can grow the scale at the entrance and shrink it at the exit and this will give a smoother transition to the animations, what do you think ? 
Demo : https://codepen.io/ardagedikk/pen/Ewbjyx

**Comment by daneden** [comment]
Hey folks, wanted to chime in here. I think that we're conflating a few things here, namely the review of two distinct animation sets: the "back in" and the "twist" animations. I quite like the "back in" animations, but I think the "twist" animations are too complex. Generally, the animations in this library ought to seem like a single, fluid motion—the twist animations are too staggered for my liking.

**Comment by ardagedikk** [comment]
I removed the twist animations. As far as I understand, there is no problem with back animations.

**Comment by ardagedikk** [comment]
?

File: animate-config.json [hunk 1]
```diff
@@ -116,5 +116,34 @@
...
+  },
+
+  "back_exits": {
+    "backOutDown": true,
+    "backOutLeft": true,
+    "backOutRight": true,
+    "backOutUp": true
+  },
+
+  "twist-entrances": {
```
Comment (eltonmesquita) [inline animate-config.json]: There are some Twist animations left overs here. Can you please fix it?

**Comment by WarenGonzaga** [comment]
any updates here? @eltonmesquita @ardagedikk @daneden ?

**Comment by eltonmesquita** [comment]
Hey @ardagedikk, any news on the fix? Also, could you please fix the conflicts? It'd be nice to have those animations on v4.

**Comment by WarenGonzaga** [comment]
Can I fix this one? @eltonmesquita so we can add this on animate.css v4?

**Comment by eltonmesquita** [comment]
Sure @WarenGonzaga. I think there would be no problem as currently @ardagedikk seems to be unavailable.

**Comment by WarenGonzaga** [comment]
@eltonmesquita should I push this one to master or into new-build?

**Comment by eltonmesquita** [comment]
new-build, please.

**Comment by eltonmesquita** [comment]
Sorry for the massive delay, but it'll get released on v4!

You can see a preview here: https://codepen.io/eltonmesquita/full/agrYzj

**Comment by eltonmesquita** [comment]
BTW it's merged on the `dev` as you can see on 793b0ffc

**Comment by WarenGonzaga** [comment]
@eltonmesquita another good job

**Comment by WarenGonzaga** [comment]
@eltonmesquita close this if necessary thanks!

**Comment by eltonmesquita** [comment]
I'm closing this as it was cherry-picked on the dev branch.

**Pull request closed without merging**